docs/mobile-apps/app-storage.md

```diff
@@ -23,6 +23,20 @@ When testing mobile apps, you have the option to upload your app to our app stor
      * [iOS Demo App](https://github.com/saucelabs/my-demo-app-ios/releases)
      * [Android Demo App](https://github.com/saucelabs/my-demo-app-android/releases)
 
+## Rate Limiting
+
+To prevent service overload and outbursts of incoming traffic, Sauce Labs introduced the Rate Limiting feature. It helps maximize the stability of incoming traffic.
```

Comment:
```suggestion
To increase service stability and prevent overload by a high volume of incoming traffic, we have set the following rate limits for uploading your mobile apps: 
```

Reply:
To prevent service overload and maximize our service quality, Sauce Labs will introduce new rate limits effective from 14.11.2022.